justfile

```diff
@@ -189,6 +195,18 @@ all: services gen native run
 # Run everything at once, tailored for iOS development (rebuilds iOS)
 all-ios: services gen ios run
 
+testnet-apk:
+    #!/usr/bin/env bash
+    BUILD_NAME=$(yq -r .version {{pubspec}})
+    BUILD_NUMBER=$(git rev-list HEAD --count)
+    echo "build name: ${BUILD_NAME}"
+    echo "build number: ${BUILD_NUMBER}"
+    ELECTRS_ENDPOINT="${ELECTRS_ENDPOINT:-$TESTNET_ELECTRS}"
+    COORDINATOR_P2P_ENDPOINT="${COORDINATOR_P2P_ENDPOINT:-$TESTNET_COORDINATOR}"
+    echo "electrs: ${ELECTRS_ENDPOINT}"
+    echo "coordinator: ${COORDINATOR_P2P_ENDPOINT}"
+    cd mobile && flutter build apk --build-name=${BUILD_NAME} --build-number=${BUILD_NUMBER} --release --dart-define="ELECTRS_ENDPOINT=${ELECTRS_ENDPOINT}" --dart-define="COORDINATOR_P2P_ENDPOINT=${COORDINATOR_P2P_ENDPOINT}"
```

Comment:
fyi: we have a github workflow for that. It's the same workflow copied from our poc.

1. draft new release: https://github.com/get10101/10101/actions/workflows/draft_new_release.yml
2. create release: once the draft release gets merged the release is created. https://github.com/get10101/10101/actions/workflows/create_release.yml

Ideally we can move the ios build to the workflow as well, eventually 🙂